src/pages/sources.vue

```diff
@@ -183,6 +183,14 @@ const SourcePage = {
 
       this.sort = { direction, field }
     },
+    cleanSourceUrlForPresentation(url) {
+      const stripProtocol = (s) => s.replace(/https?:\/\//, '')
+      const stripLeadingWww = (s) =>
+        s.startsWith('www.') ? s.replace('www.', '') : s
+      const stripTrailingSlash = (s) => (s.endsWith('/') ? s.slice(0, -1) : s)
+
+      return stripTrailingSlash(stripLeadingWww(stripProtocol(url)))
+    },
```

Comment:
I see you planting the seeds of some FP in the codebase 😉. We'll refactor this to `pipe(stripTrailingSlash, stripLeadingWww, stripProtocol)(url)` someday!

Reply:
One day!